notebooks/halos_rendering.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>You don't need this line: "The first line of code is an import statement, it brings in the <code>HalosSkyPyPipeline</code> class from the <code>sim_pipeline.Pipelines.halos_pipeline</code> module." It is self-evident just by looking at the code.</p><p> </p><p>Instead, you can say something like "We first create an instance of the <code>HalosSkyPyPipeline</code> ..." and then explain why these values for <code>m_max</code>,<code>m_min</code>, etc. are chosen.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sibirrer/sim-pipeline/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>